### PR TITLE
Added reselect@2.5.1

### PR DESCRIPTION
**Pull request by ronnyroeller**
Fixes https://github.com/cdnjs/cdnjs/issues/8372

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/reactjs/reselect
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/reselect
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3,715
   - Count of watchers: 179
   - Count of forks: 2,627
 * NPM download stats (optional):
   - Downloads in the last day: 1,374
   - Downloads in the last week: 33,632
   - Downloads in the last month: 130,158

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/reselect/package.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  "license": "MIT",
+  "npmName": "reselect",
+  "npmFileMap": [
+    {
+      "basePath": "/src",
+      "files": [
+        "**/*"
+      ]
+    }
```
Comment (PeterDaveHello) [inline ajax/libs/reselect/package.json]: I think we should use `dist` as base path.

**Comment by PeterDaveHello** [comment]
@ronnyroeller would you please fix the errors? Some of the useless sections in package.json should be removed, thanks for your contribution.

File: ajax/libs/reselect/package.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  ],
+  "filename": "/index.js"
```
Comment (kennynaoh) [inline ajax/libs/reselect/package.json]: I think `filename` should be `reselect.min.js`.
Thanks!

**Comment by PeterDaveHello** [comment]
Hello @ronnyroeller, would you like to update this PR? Thanks.

**Comment by ronnyroeller** [comment]
No longer required

**Pull request closed without merging**